code/src/ThoughtBox.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import moment from "moment";
```

Comment:
Separate these sections of dependencies with one space, as you have with the other components.